src/main/java/com/marklogic/appdeployer/command/databases/DeployDatabaseCommand.java

```diff
@@ -97,6 +97,8 @@ public void execute(CommandContext context) {
         String payload = buildPayload(context);
         if (payload != null) {
             DatabaseManager dbMgr = new DatabaseManager(context.getManageClient());
+
+            payload = adjustPayloadBeforeSavingResource(dbMgr, context, null, payload);
```

Comment:
Thoughts on this - is it worth combining these two lines of code? They appear in several of these classes, along with AbstractCommand. 

Perhaps:

    protected SaveReceipt saveResource(...) {
      adjustPayload...
      mgr.save(payload)
      if (storeResourceIdAsCustomTokens)...
      return saveReceipt
    }

Reply:
The benefit being that we cut down on the calls to adjustPayloadBeforeSavingResource .